subprojects/docs/src/docs/userguide/standardPlugins.adoc

```diff
@@ -121,10 +121,20 @@ These plugins provide some integration with various runtime technologies.
 | `java`
 | Adds support for building J2EE applications.
 
+| <<publishing_ivy, `ivy-publish` >>
+| -
+| `application`, `distribution`, `java`, `war`
+| Provides a new DSL to support publishing artifacts to Ivy repositories, which improves on the existing DSL.
+
+| <<publishing_maven, `maven-publish` >>
+| -
+| `application`, `distribution`, `java`, `war`
+| Provides a new DSL to support publishing artifacts to Maven repositories, which improves on the existing DSL.
+
```

Comment:
This would ideally be a definition list, but this can be changed at a later date.

Reply:
There are a lot of tables in this chapter so I'll leave it as-is for now.